src/Card.js

```diff
@@ -0,0 +1,10 @@
+class Card {
+  constructor(id, question, answers, correctAnswer) {
+    this.id = id;
+    this.question = question;
+    this.answers = answers;
```

Comment:
I needed to make sure this property was named correctly in order to work with the Card-test.js file. Passes all tests now!